src/index.js

```diff
@@ -4,9 +4,18 @@ import ReactDOM from "react-dom";
 import "./styles.css";
 import { Game } from "./Game";
 
-const App = function() {
-  const boardSizeX = 10;
-  const boardSizeY = 10;
+import getUrlParams from './getUrlParams';
+
+const App = function () {
+
+  let size = parseInt(getUrlParams('size'), 10);
```

Comment:
Use the in-built UrlSearchParams function, good article about it here https://javascriptplayground.com/url-search-params/

Reply:
Modified.  Thank you for sharing the link.  I learned something new!